core/src/main/scala/chisel3/Layer.scala

```diff
@@ -129,11 +129,12 @@ object layer {
     * layerblock is not an ancestor of the desired layer
     */
   def block[A](
-    layer: Layer
-  )(thunk: => A
+    _layer: Layer
+  )(thunk:  => A
```

Comment:
```suggestion
    layer: Layer
  )(thunk:  => A
```
I would keep this the same and use a different name for the value below. The argument is part of the public API (named arguments and ScalaDoc), and thus I think it's better to keep it as the nicer argument name.

Reply:
Good ABI catch! Fixed in 2778c92f0.